selfdrive/car/toyota/values.py

```diff
@@ -298,23 +298,39 @@ class CAR:
 
 FW_VERSIONS = {
   CAR.AVALON: {
-    (Ecu.esp, 0x7b0, None): [b'F152607060\x00\x00\x00\x00\x00\x00'],
+    (Ecu.esp, 0x7b0, None): [
+      b'F152607060\x00\x00\x00\x00\x00\x00',
```

Comment:
why are there multiple fw versions added here?

Reply:
without those two it goes into dashcam

Reply:
I checked three routes and am consistently seeing `F152607180\x00\x00\x00\x00\x00\x00`. remove the other one

Reply:
if you'd like to add more FW, do it in another PR